p2p/simulations/network.go

```diff
@@ -388,7 +399,7 @@ func (self *Network) Stop(id discover.NodeID) error {
 	node.Up = false
 	log.Info(fmt.Sprintf("stop node %v: %v", id, node.Up))
 
-	self.events.Send(ControlEvent(node))
+	self.events.Send(NewEvent(node))
```

Comment:
Did you figure out what was wrong with the previous attempt to add live stop events? I don't think this change here is correct.